blocks_vertical/data.js

```diff
@@ -486,6 +486,40 @@ Blockly.Extensions.registerMixin('contextMenu_getVariableBlock',
   Blockly.Constants.Data.CUSTOM_CONTEXT_MENU_GET_VARIABLE_MIXIN);
 
 /**
+ * Mixin to add a context menu for a data_list block.  It adds one item for
+ * each list defined on the workspace.
+ * @mixin
+ * @augments Blockly.Block
+ * @package
+ * @readonly
+ */
+Blockly.Constants.Data.CUSTOM_CONTEXT_MENU_GET_LIST_MIXIN = {
```

Comment:
That makes more sense. Alright, will revert to before I added the context menu mixin for lists.